DashAI/back/core/schema_fields/utils.py

```diff
@@ -0,0 +1,44 @@
+from dependency_injector.wiring import Provide, inject
+
+from DashAI.back.core.schema_fields.base_schema import BaseSchema
+
+
+@inject
+def fill_objects(
+    schema_instance: BaseSchema,
+    component_registry=Provide["component_registry"],
+) -> dict:
+    """Fills in the schema instance, replacing the component fields with the
+    target component. Returns the dumped dictionary of the schema instance.
+
+    This function transforms all fields of the component into actual components.
+    To do this, the component type is looked up in the component registry and
+    instantiated using the corresponding parameters.
+
+    The function is called recursively on component parameters, to fill its
+    internal components.
+
+    Parameters
+    ----------
```

Comment:
This function looks a bit complex. 
Could you write an example in the docstring (similar to the one in the component registry?)

```suggestion
    The function is called recursively on component parameters, to fill its
    internal components.
    
    Example
    ---------
    ...

    Parameters
    ----------
```